registry/registry-nacos/src/main/java/com/alipay/sofa/rpc/registry/nacos/NacosRegistryHelper.java

```diff
@@ -83,7 +83,7 @@ static List<Instance> convertProviderToInstances(ProviderConfig providerConfig)
 
                 // set host port
                 String host = server.getVirtualHost();
-                if (host == null) {
+                if (StringUtils.isEmpty(host)) {
```

Comment:
Simple test case needs to be added

Reply:
It is the code in Nacos registry module, how about other places related to `host`?  I have searched the source code , many places are using `host == null` to determine whether host is empty. We should modify all of them. @EvenLjj @Lo1nt